proxy/Dockerfile

```
@@ -0,0 +1,24 @@
+FROM python:3.9.16-slim-buster
+
+WORKDIR /usr/src/app
+
+# set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+USER 0:123
+RUN apt-get -y update
+RUN apt-get install -y iptables
+RUN apt-get install -y tcpdump
```

nit: I think this will use fewer layers / make for a slightly more optimal build
```suggestion
RUN apt-get -y update && apt-get install -y iptables tcpdump
```